config.go

```diff
@@ -286,14 +286,14 @@ type spriteConfig struct {
 	AnimBindings        map[string]string     `json:"animBindings"`
 	CollisionMask       *int64                `json:"collisionMask"`
 	CollisionLayer      *int64                `json:"collisionLayer"`
-	ColliderShapeType   string                `json:"colliderShapeType"`
-	ColliderPivot       mathf.Vec2            `json:"colliderPivot"`
+	CollisionShapeType  string                `json:"collisionShapeType"`
+	CollisionPivot      mathf.Vec2            `json:"collisionPivot"`
 	// ColliderShapeParams defines the shape parameters based on ColliderShapeType:
 	// - "Rect": [width, height] - Rectangle with specified width and height
 	// - "Circle": [radius] - Circle with specified radius
 	// - "Capsule": [radius, height] - Capsule with specified radius and height
 	// - "Polygon": [x1, y1, x2, y2, ...] - Polygon with vertex coordinates in pairs
-	ColliderShapeParams []float64  `json:"colliderShapeParams"`
+	ColliderShapeParams []float64  `json:"collisionShapeParams"`
```

Comment:
这里的字段名 `ColliderShapeParams` 也应该调整？


```suggestion
	CollisionShapeParams []float64  `json:"collisionShapeParams"`
```

Reply:
Fixed